docs/tutorial/index.md

```diff
@@ -455,62 +293,50 @@ Hit:5 http://ppa.launchpad.net/landscape/self-hosted-beta/ubuntu noble InRelease
 
-
-- To accept the registration, click on the instance name; in the pop-up, set "Tags" to  `wsl-vision`; finally, click **Accept**.
+To accept the registration click on the instance name, set "Tags" to `wsl-vision` in the pop-up then click **Accept**.
+The `wsl-vision` tag will be used for all the instances accepted into Landscape.
 
 ![Accept and tag Ubuntu Preview](./assets/accept-ubuntu-preview-tag.png)
 
+### Create an Ubuntu WSL instance remotely
 
-### Use Landscape to create a pro-attached and Landscape-registered Ubuntu WSL instance remotely
-
-Back to your Windows browser, at the Landscape page, navigate to "Computers" and click on the Windows machine (below, `mib`). You'll find the "WSL Instances" on the right-hand side of the page and an "Install new" link close to it.
-Click on that link. Once the page has loaded, set "Instance Type" to "Ubuntu", then click "Submit". A status page will
+Back on the Landscape page in your web browser, navigate to "Computers" and click on the Windows machine (below: `mib`). You will find "WSL Instances" on the right side of the page.
+Click on the "Install new" link then set "Instance Type" to "Ubuntu" and click "Submit". A status page will
 appear showing the progress of the new instance creation.
 
 ![Create instance via Landscape](./assets/create-instance-via-landscape.png)
 
 ![Creation progress](./assets/creation-progress.png)
 
+The Landscape server will talk to the Landscape client built into your UP4W.
+UP4W will then install the `Ubuntu` application and create an Ubuntu WSL instance automatically.
+In PowerShell, run `ubuntu.exe` to log in to the new instance.
 
-Your Landscape Server will talk to the Landscape client built into your UP4W and ask UP4W to install the `Ubuntu` application and create an Ubuntu WSL instance for you. In your PowerShell, run `ubuntu.exe` to log in to the new instance.
-
-```powershell
+```text
 PS C:\Users\me\tutorial> ubuntu.exe
-To run a command as administrator (user "root"), use "sudo <command>".
-See "man sudo_root" for details.
-
-Welcome to Ubuntu 22.04.3 LTS (GNU/Linux 5.15.146.1-microsoft-standard-WSL2 x86_64)
-
- * Documentation:  https://help.ubuntu.com
- * Management:     https://landscape.canonical.com
- * Support:        https://ubuntu.com/advantage
 
-
-This message is shown once a day. To disable it please create the
-/home/me/.hushlogin file.
 me@mib:~$
 ```
 
-As usual, this instance is pro-attached and Landscape-registered -- run `pro status` to verify the pro-attachment and refresh your Landscape server page to verify and accept the registration (as before, apply the  `wsl-vision` tag and click `Accept`).
-
+You can run `pro status` to verify pro-attachment and refresh your Landscape server page to verify and accept the registration.
+As before, apply the `wsl-vision` tag and click `Accept`.
 
-### Use Landscape to deploy packages to all of your Ubuntu WSL instances at once
+### Deploy packages to all Ubuntu WSL instances
```

Comment:
As before, the previous heading was meant to have some kind of marketing effect: if the reader is not familiar with Landscape, that title would immediately show off one of its core values. With the proposed changes we still show the value, but detach it from the Landscape product. No strong opinion here, but I'd like to make sure you considered that effect when doing the changes. WDYT?